lib/spack/spack/cmd/install.py

```diff
@@ -106,6 +107,8 @@ def setup_parser(subparser):
         '--cache-only', action='store_true', dest='cache_only', default=False,
         help="only install package from binary mirrors")
 
+    monitor_group = spack.monitor.get_monitor_group(subparser)  # noqa
```

Comment:
does this need noqa?

Reply:
Yes, if I don't have it, flake8 tells me there is an unused variable.

Reply:
I can take out the variable and add a comment, that should work and not need the noqa. I left it there because I want the developer to know what the heck it does.